container/default.conf

```diff
@@ -0,0 +1,31 @@
+map $sent_http_content_type $expires {
+    default                    off;
+    text/html                  epoch;
+    text/css                   1d;
+    application/javascript     1d;
+    ~image/                    1d;
+}
+
+server {
+    listen       80;
+    server_name  localhost;
+
+    expires $expires;
+
+    #charset koi8-r;
+    #access_log  /var/log/nginx/host.access.log  main;
+
+    location / {
+        root   /usr/share/nginx/html;
+        index  index.html index.htm;
+    }
+
+    #error_page  404              /404.html;
```

Comment:
I think this should work with Hugo: https://gohugo.io/templates/404/#automatic-loading

We have an empty theme file for it currently, but I'll push this in a second. `/404.html` now works for me locally.

Reply:
I was intending to ask about that once this one was merged...

Reply:
https://github.com/ScilifelabDataCentre/covid-hub/pull/28 open now, so I think that you can probably just uncomment this here if we merge both 👍